cli/aws_orbit/models/context.py

```diff
@@ -290,20 +290,15 @@ def fetch_env_data(self) -> None:
 
     def fetch_cognito_external_idp_data(self) -> None:
         _logger.debug("Fetching Cognito External IdP data...")
-        if self.cognito_external_provider:
```

Comment:
This conditional was added to avoid writing to context default okta config that will make the deployment think okta integration is active when it isnt